From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @chipalost, has been notified to review your proposed change.

4c74356b41 commented:
Just a ping ;)

dominicbetts commented:
@4c74356b41 - it looks like there's now a merge conflict on this PR - can you fix the merge conflict and then we can accept your change?
Thanks, Dominic

dominicbetts commented:
#please-close
@4c74356b41 @chipalost - rather than you fixing the merge conflict - I've made the change directly.

tynevi committed:
index and toc yml's

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into ssh

Commit:
commit

Commit:
commit-1

mrbullwinkle committed:
modify ms.author metadata

dlepow committed:
consistency edits

rayne-wiselman committed:
Update toc.yml

rayne-wiselman committed:
Update migrate-scenarios-overview.md

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#7233

Commit:
PR comments

j-martens committed:
AML: adding templates for PM partners

rayne-wiselman committed:
Update migrate-scenarios-assessment.md

barbkess committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into topic-updates3

barbkess committed:
fix link

dlepow committed:
missed edits

tynevi committed:
images for index

iainfoulds committed:
[VM] Additional edits

tynevi committed:
clean up redirects

douglaslMS committed:
Other revisions from PM related to SSIS Activity.

barbkess committed:
metadata updates

v-shils committed:
Merge pull request #37084 from srrengar/sravnewarticles

OMS analyze instructions

Commit:
fix for git issue 7074

PRMerger14 committed:
Merge pull request #38129 from iainfoulds/vmadminusername

[VM] Clarifying use of --admin-username

barbkess committed:
acrolinx fixes

PRMerger16 committed:
Merge pull request #38128 from mrbullwinkle/mrb_04_17_2018_author_metadata

modify ms.author metadata

Commit:
automation : update start stop tutorial

Commit:
updates

tynevi committed:
Merge pull request #38133 from tysonn/release-cloudml-refactor2

CloudML refactor - moving articles to service folder

v-shils committed:
Merge pull request #37862 from rayne-wiselman/rayne-migration-scenario-april14

adding migration scenarios for review

barbkess committed:
metadata updates

PRMerger19 committed:
Merge pull request #38067 from oslake/patch-4

Update sql-database-dtu-resource-limits.md

rjagiewich committed:
Merge pull request #37753 from v-nagta/fiscalnote

USER STORY 1235481 SaaS App Tutorial: FiscalNote

carolinacmoravia committed:
M16626: Possible incorrect term

Hello, @philmea, 

Translator has reported possible source content issue.   

Please, help to check my proposed file change into the article and help to merge if you agree with fix.  not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

v-shils committed:
Merge pull request #37140 from ggailey777/current

Update install requirements for Core Tools

GitHubber17 committed:
Merge pull request #38003 from SnehaGunda/cdbthroughput

Moving MongoDB specific throughput details

GitHubber17 committed:
Merge pull request #38034 from JasonWHowell/sa-deletes

Article deletes for stream analytics

v-shils committed:
Merge pull request #37817 from barbkess/topic-updates3

update provision-powershell

GitHubber17 committed:
Update azure-to-azure-about-networking.md

navyasric committed:
Add Python auth sample links

ajlam committed:
Update MySQL and PG gen5

GitHubber17 committed:
Merge pull request #38118 from davidmu1/gitissue7049

fix for git issue 7049

mimckitt committed:
Fixes MicrosoftDocs/azure-docs#7280

The UPN format should be, user_contoso.com#EXT#@fabrikam.onmicrosoft.com (with two # before and after EXT) instead of user_contoso.com#EXT@fabrikam.onmicrosoft.com (with only one #)

daveba committed:
updating TOC and removing articles

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into remove-storage-articles

anavinahar committed:
Additional regions for 4/18 GA

Americas: West Central US (Wyoming), West US 2 (Washington), Central US (Iowa), US East 2 (Virginia), Canada Central (Toronto), Canada East (Quebec City)
Asia Pacific: Korea South (Buscan), Korea Central (Seoul), South India (Chennai), Central India (Pune), West India (Mumbai)
Europe: UK South (London), UK West (Cardiff)

GitHubber17 committed:
Merge pull request #38113 from sujayt/master

nsg-guidance update

anzaman committed:
Update vpn-gateway-about-point-to-site-routing.md

Commit:
updates

v-geberr committed:
new screen shots with zoom

PRMerger18 committed:
Merge pull request #38147 from anzaman/patch-2

Update vpn-gateway-about-point-to-site-routing.md

v-shils committed:
Merge pull request #38135 from barbkess/topic-updates11

metadata updates

chetansriv committed:
Update AWS multi account

PRMerger18 committed:
Merge pull request #38145 from MicahMcKittrick-MSFT/patch-19

Fixes MicrosoftDocs/azure-docs#7280

barclayn committed:
making changes based on reviewer's feedback

GitHubber17 committed:
Merge pull request #38122 from cynthn/reassignments

reassigning topics

PRMerger19 committed:
Merge pull request #38149 from ajlam/master

 Update MySQL and PG gen 5 regions

daveba committed:
updating redirect

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into remove-storage-articles

GitHubber17 committed:
Merge pull request #35359 from hareeshnagaraj/pr/1

A new documentation for diagnostic Reliable State Manager / Reliable Collections performance using the performance counters.

CarlRabeler committed:
Update sql-database-what-is-a-dtu.md

v-shils committed:
Merge pull request #38151 from v-geberr/0417-how-to-create-new-app-screenshots

new screen shots with zoom

markgalioto committed:
adding link to Backup powershell to fix GH Issue

chetansriv committed:
Update AWS multi account

KumudD committed:
fixed Acroliins issues

KumudD committed:
fixed Acrolinx issues

chetansriv committed:
Updating custom app role tutorial

v-shils committed:
Merge pull request #38139 from barbkess/topic-updates12

metadata updates

tamram committed:
clarification for SSE per issue 5519

PRMerger10 committed:
Merge pull request #38154 from CarlRabeler/patch-51

Update sql-database-what-is-a-dtu.md

PRMerger12 committed:
Merge pull request #38138 from georgewallace/startstop

automation : start stop tutorial update to reflect pricing changes

Commit:
ran acrolinx and stepped through

chetansriv committed:
Merge branch 'master' into customrole

billmath committed:
removing duplicate

j-martens committed:
AML: fixing broken links in new TOC

martincoetzer committed:
Update azure-ad-choose-authn.md

Fixing project plan link

v-shils committed:
Merge pull request #37576 from barclayn/ddos

adding the ddos article

markgalioto committed:
editing the article

PRMerger14 committed:
Merge pull request #38158 from tamram/tamram-0417a

clarification for SSE per issue 5519

KumudD committed:
addressed Acrolinx issues

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into filerename0416

v-shils committed:
Merge pull request #38110 from cephalin/escape

add --%

PRMerger16 committed:
Merge pull request #38161 from martincoetzer/patch-3

Update azure-ad-choose-authn.md

j-martens committed:
AML: Fixing broken links in directory renaming

PRMerger16 committed:
Merge pull request #38155 from markgalioto/apr172018

adding link to Backup powershell to fix GH Issue

CarlRabeler committed:
[sql db] spark connector

GitHubber17 committed:
Merge pull request #38160 from billmath/pimfix

removing duplicate

daveba committed:
updating redirect

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into remove-storage-articles

CarlRabeler committed:
more

CarlRabeler committed:
Update sql-database-performance-guidance.md

j-martens committed:
AML: last broken link

v-shils committed:
Merge pull request #38137 from davidmu1/gitissue7074

fix for git issue 7074

GitHubber17 committed:
Merge pull request #38143 from navyasric/link_auth_sample

Add Python auth sample links

GitHubber17 committed:
Merge pull request #38086 from KumudD/filerename0416

File renaming and redirection

Commit:
Merge pull request #38165 from CarlRabeler/patch-52

Update sql-database-performance-guidance.md

GitHubber17 committed:
Merge pull request #38146 from daveba/remove-storage-articles

[Azure AD MSI] Remove storage articles

douglaslMS committed:
Removed offending ARM acronym.

JasonWHowell committed:
new overview

j-martens committed:
AML: fixing broken links found in renaming effort

billmath committed:
updating per hotfix release

barbkess committed:
Casey Karst - author

Ja-Dunn committed:
Merge pull request #38167 from dan-wesley/dawesl-docrefresh-apr17

ran acrolinx and step through

Ja-Dunn committed:
Merge pull request #38134 from douglaslMS/adf-ssis-other2

Other revisions from PM related to SSIS Activity.

PRMerger17 committed:
Merge pull request #38072 from MohitGargMSFT/patch-12

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#7195

PRMerger19 committed:
Merge pull request #38174 from billmath/verhist

updating per hotfix release

conceptdev committed:
remove 2013, add 2017 instructions

barbkess committed:
kavitha author

Duncanma committed:
Removing review.docs link

j-martens committed:
Adding missing image back again.

Ja-Dunn committed:
Merge pull request #38172 from JasonWHowell/screenshot

Higher res image for the overview

mumian committed:
acrolinx

Commit:
removed review from images links

GitHubber17 committed:
Merge pull request #37421 from jimdial/ddos-new

DDoS new

barbkess committed:
jrj-author

ktoliver committed:
Merge pull request #37441 from chetansriv/customrole

Customrole

ktoliver committed:
Merge pull request #37635 from chetansriv/awsmultiaccount

AWS Multiple Accounts Tutorial

Commit:
Merge pull request #38180 from MicrosoftDocs/FixNetworkWatcherPage

Removing review.docs link

Ja-Dunn committed:
Merge pull request #38140 from carolinacmoravia/patch-680

M16626: Possible incorrect term

Ja-Dunn committed:
Merge pull request #33098 from rapatchi/redhat-setup

Redhat setup

barbkess committed:
update authors

Ja-Dunn committed:
Merge pull request #37935 from mumian/0416-meta-spark

[hdi] update meta of the spark articles

barbkess committed:
acrolinx fixes

Brenduns committed:
fixing note

Commit:
Task 985: Fix security token/SAS confusion in option 3

Ja-Dunn committed:
Merge pull request #38183 from davidmu1/indexcorrection

removed review from images links

tynevi committed:
Merge pull request #38173 from j-martens/release-cloudml-refactor2

AML: fixing broken links for renaming effort

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

omidm1 committed:
Updating Author

Ja-Dunn committed:
Merge pull request #38187 from barbkess/igor-authors2

update authors

anavinahar committed:
adding text for Global VNet peering

aashishb committed:
added back the step to register Microsoft.ContainerRegistry

anavinahar committed:
Update virtual-network-peering-overview.md

tynevi committed:
resolving mc in redirect file

tynevi committed:
Merge pull request #38199 from tysonn/release-cloudml-refactor2

Merging master to release branch as part of conflict resolution

v-alje committed:
Merge pull request #38190 from MicrosoftDocs/FromPublicRepo

From public repo

ktoliver committed:
Merge pull request #36869 from dlepow/ssh

Linux VMs: Freshen SSH content

rmca14 committed:
Merge pull request #37266 from paulshealy1/master

Update Ubuntu docs for the latest release

anavinahar committed:
Update virtual-network-peering-overview.md

tfitzmac committed:
clarified delivery

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo

subsarma committed:
Update dns-alerts-metrics.md

Added to the note on no support for sovereign clouds

anavinahar committed:
Update virtual-network-manage-peering.md

ktoliver committed:
Merge pull request #38176 from barbkess/igor-authors

Casey Karst - author

anavinahar committed:
Update virtual-network-manage-peering.md

PRMerger14 committed:
Merge pull request #38192 from anavinahar/patch-44

adding text for Global VNet peering

anavinahar committed:
Update virtual-network-manage-peering.md

anavinahar committed:
Update virtual-network-manage-peering.md

tynevi committed:
Merge pull request #38130 from j-martens/master

AML: adding templates for PM partners

tfitzmac committed:
fix

aashishb committed:
Merge branch 'master' into master

PRMerger18 committed:
Merge pull request #38189 from Brenduns/quick-fix

fixing note

j-martens committed:
AML: fix broken links in index file

PRMerger18 committed:
Merge pull request #38193 from omidm1/master

Updating Author

PRMerger19 committed:
Merge pull request #38181 from conceptdev/patch-3

[mobile apps] remove 2013, add 2017 instructions

Commit:
Merge pull request #38202 from tfitzmac/0417delivery

clarified delivery

mattbriggs committed:
adding verbatim updates

Commit:
Merge pull request #38203 from subsarma/patch-4

Update dns-alerts-metrics.md

rmca14 committed:
Merge pull request #38164 from CarlRabeler/spark

[sql db] spark connector

CarlRabeler committed:
Update sql-database-read-scale-out.md

CarlRabeler committed:
Update sql-database-read-scale-out.md

martincoetzer committed:
Updating arrows

omidm1 committed:
Updated Author

ktoliver committed:
Merge pull request #37897 from v-nagta/bpmonline

USER STORY 1237011 SaaS App Tutorial: Bpmonline Metadata URL update

PRMerger14 committed:
Merge pull request #38196 from aashishb/master

added back the step to register Microsoft.ContainerRegistry

v-alje committed:
Bringing even with master.

markgalioto committed:
fixing Backup VM article so it is current with Azure portal

PRMerger16 committed:
Merge pull request #38212 from CarlRabeler/patch-53

Update sql-database-read-scale-out.md

ktoliver committed:
Merge pull request #37900 from v-nagta/compliance

USER STORY 1237019 SaaS App Tutorial: Compliance ELF Metadata URL update

rmca14 committed:
Merge pull request #37898 from v-nagta/cernercentral

USER STORY 1237013 SaaS App Tutorial: Cerner Central Metadata URL update

PRMerger17 committed:
Merge pull request #38216 from omidm1/master

Updated Author

ktoliver committed:
Merge pull request #37902 from v-nagta/contractworks

User Story 1237021: SaaS App Tutorial: ContractWorks Metadata URL update

ktoliver committed:
Merge pull request #38214 from martincoetzer/master

Updating arrows

rmca14 committed:
Merge pull request #37903 from v-nagta/formcom

User Story 1237022: SaaS App Tutorial: Formcom Metadata URL update

mrbullwinkle committed:
Add reference to Azure PowerShell and REST API

kevinvngo committed:
Merge branch 'master' into patch-24

Commit:
Task 985: Add regex example

markgalioto committed:
fixing broken link

Ja-Dunn committed:
Merge pull request #36553 from lisawong19/cog-speech

re-arranged TOCs & re-named "Get Started" to Quickstarts

Juliako committed:
Update media-services-release-notes.md

Juliako committed:
Update media-services-java-how-to-use.md

Ja-Dunn committed:
Merge pull request #37548 from jingyanjingyan/spark3vscode

spark3 in vscode

Ja-Dunn committed:
Merge pull request #38198 from dksimpson/dean-985

Task 985: Fix security token/SAS confusion in option 3

Ja-Dunn committed:
Merge pull request #38206 from j-martens/release-cloudml-refactor2

AML: fix broken links in index file

Ja-Dunn committed:
Merge pull request #38211 from mattbriggs/mdb_20180417_verbatim_checks

adding verbatim updates

Ja-Dunn committed:
Merge pull request #38219 from mrbullwinkle/mrb_04_17_2018_reference_azure_powershell

Add reference to Azure PowerShell and REST API

markgalioto committed:
more updates to fix outdated art

genlin committed:
resolve conflict

Ja-Dunn committed:
Merge pull request #37890 from genlin/990

reset-rdp.md

PRMerger18 committed:
Merge pull request #38233 from Juliako/patch-12

Update media-services-java-how-to-use.md

PRMerger18 committed:
Merge pull request #38232 from Juliako/patch-11

Update media-services-release-notes.md

Commit:
update structure

rwike77 committed:
auto scale article

rwike77 committed:
typo

Sogup committed:
Update backup-azure-recovery-services-vault-overview.md

Sogup committed:
Update backup-introduction-to-azure-backup.md

Sogup committed:
Update azure-backup-limits.md

rwike77 committed:
formatting changes

PRMerger17 committed:
Merge pull request #38243 from Sogup/patch-59

Update azure-backup-limits.md

PRMerger20 committed:
Merge pull request #38242 from Sogup/patch-58

Update backup-introduction-to-azure-backup.md

rayne-wiselman committed:
updating table content

Juliako committed:
Update api-management-howto-deploy-multi-region.md

PRMerger18 committed:
Merge pull request #38247 from rayne-wiselman/rayne-migrate-scenarios-april18

updating table content

PRMerger20 committed:
Merge pull request #38248 from Juliako/patch-11

Update api-management-howto-deploy-multi-region.md

cephalin committed:
MicrosoftDocs/azure-docs#7054

PRMerger14 committed:
Merge pull request #38240 from Sogup/patch-56

Update backup-azure-recovery-services-vault-overview.md

hrushib committed:
Minor fixes in existing backup restore document to make it's Acrolinx score more than 80.

PRMerger20 committed:
Merge pull request #34189 from BogdanMarkovic/patch-1

Update of public documentation related to FD/UDs

PRMerger12 committed:
Merge pull request #38250 from cephalin/issue7054

MicrosoftDocs/azure-docs#7054

msvijayn committed:
cleanup reduntant code in sample

PRMerger20 committed:
Merge pull request #38253 from msvijayn/vijayn-docs

cleanup reduntant code in sample

PRMerger17 committed:
Merge pull request #35220 from carolinacmoravia/patch-646

CC51959: Extra content that might be deleted

Sogup committed:
Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

Commit:
automation: update image in vm solution article

Sogup committed:
Update backup-azure-arm-vms-prepare.md

PRMerger17 committed:
Merge pull request #38256 from Sogup/patch-60

Update backup-azure-troubleshoot-vm-backup-fails-snapshot-timeout.md

Commit:
automation: correcting metadata

PRMerger19 committed:
Merge pull request #38260 from Sogup/patch-65

Update backup-azure-arm-vms-prepare.md

douglaslMS committed:
One minor fix from PM.

Commit:
Merge pull request #38148 from anavinahar/patch-43

Additional regions for 4/18 GA

PRMerger10 committed:
Merge pull request #38259 from georgewallace/master

automation: update image in vm solution article

hrushib committed:
Updating toc to reference standalone article.
Updating Drop down names to be cosistent with the ones used in TOC.

carolinacmoravia committed:
CC53358: Possible incorrect pictures

Hi, @cephalin,
This contirbution comes from https://github.com/MicrosoftDocs/azure-docs.es-es/pull/95.
Could you help to review and merge this PR, please?
Many thanks in advance.

rjagiewich committed:
Merge pull request #37893 from dominicbetts/fix-jobs

Remove incorrect mention of jobs

Brenduns committed:
get support statement added

PRMerger12 committed:
Merge pull request #38263 from carolinacmoravia/patch-683

CC53358: Possible incorrect pictures

itechedit committed:
Merge pull request #38047 from kevinvngo/patch-24

removed Info on server level auditing

martincoetzer committed:
Update azure-ad-choose-authn.md

Remove redundant word "diagram" after tree

PRMerger16 committed:
Merge pull request #38267 from martincoetzer/patch-4

Update azure-ad-choose-authn.md

itechedit committed:
Merge pull request #38085 from jingyanjingyan/adlvscode

adl vscode

msvijayn committed:
add note

CarlRabeler committed:
authors

rjagiewich committed:
Merge pull request #38217 from markgalioto/417b

fixing Backup VM article so it is current with Azure portal

neilpeterson committed:
[AKS] jenkins service restart no longer needed

ggailey777 committed:
Clarified that the trigger is invoked for every change in the feed

PRMerger13 committed:
Merge pull request #38272 from ggailey777/patch-12

Clarified that the trigger is invoked for every change in the feed

PRMerger13 committed:
Merge pull request #38266 from Brenduns/quick-fix

get support statement added

Commit:
changing AzurePS cmdlet to Connect-AzureRmAccount

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger16 committed:
Merge pull request #38274 from neilpeterson/aks-jenkins-patch

[AKS] jenkins service restart no longer needed

hrushib committed:
Removing line talking about future support.

rjagiewich committed:
Merge pull request #38239 from rwike77/autoscale

Autoscale article

tynevi committed:
Merge pull request #38277 from sdwheeler/sdw-fixazlogin

changing AzurePS cmdlet to Connect-AzureRmAccount

rmca14 committed:
Merge pull request #37303 from hrushib/servicefabric-periodic-backuprestore

Servicefabric periodic backuprestore

v-alje committed:
Merge pull request #38278 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Merge pull request #38276 from CarlRabeler/master

authors

tynevi committed:
Merge pull request #38188 from MicrosoftDocs/release-cloudml-refactor2

Release cloudml refactor2 test

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo

Commit:
Merge pull request #38271 from msvijayn/vijayn-docs

add note

Commit:
Merge pull request #38261 from douglaslMS/adf-fix-ir

One minor fix from PM.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
